atoma-service/src/handlers/chat_completions.rs

```diff
@@ -153,12 +154,16 @@ pub async fn chat_completions_handler(
         "Received chat completions request, with payload hash: {payload_hash:?}"
     );
 
-    // Check if streaming is requested
     let is_stream = payload
         .get(STREAM_KEY)
         .and_then(|s| s.as_bool())
         .unwrap_or_default();
-    let endpoint = request_metadata.endpoint_path.clone();
+    let endpoint = endpoint_path.clone();
+
+    let client_public_key = client_encryption_metadata
```

Comment:
I opted to use the client's public key here as a unique identifier to track successful / failed requests (as opposed to the model given multiple nodes may run the same model)  but I think we may want to also track this with some kind of public ULID per node as well. Thoughts?

Reply:
I would say that is best to just use `model`, the reason being these are metrics for each node individually. So we might just decouple this in terms of model alone.

Reply:
Makes sense given a node can run multiple models, thanks for clarifying

updated in a383ece